_includes/tell-me-why/manage-notifications.md

```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default, notifications will be send by email when there is activity in the thread.  
+You can also choose to receive notifications of a full repository. Whenever there is a new _issue_, _pull request_ or _comment_ posted you will get a notification.
+
+###You can silence or unmute notifications of _individual issues and pull requests_.
+
+1. Go to the issue or pull request.
+2. Click the _(Un)subscribe_ button on the right, under _Notifications_  
+Underneath the button a short description explains the current notification status.
+
+###You can _customize_ notifications in Settings.
+
+1. Go to your profile icon
+2. Go to _Settings_
+3. Select _Notifications_ from the menu on the left.  
+Here you can adjust your notification [preferences](https://help.github.com/articles/managing-notification-delivery-methods/). 
+
+###You can choose to receive notifications from a _repository_.
+
+* _**Watch**_ : You will receive a notifications when a **new** _issue_, _pull request_ or _comment_ is posted, and when an _issue_ is **closed** or a _pull request_ is **merged**.  
+* _**Not watching**_ : Stop receiving notifications, but @ mentions will still alert you.  
+* _**Ingore**_ : Stops all notifications.  
+
+###You can _review notifications_ for the repositories you are already watching.
+
+1. Go to your profile icon
+2. Go to _settings_
+3. Select _Notification_ from the menu on the left.
+4. Click on the [repositories you’re watching](https://github.com/watching) link.
+5. Select the _Watching_ tab.
+6. Click the _(Un)watch_ button to disable or enable notifications for that repository.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ocQldxF7fM" frameborder="0" allowfullscreen></iframe>
```

Comment:
This video doesn't work for me 

![screen shot 2017-02-21 at 3 28 14 pm](https://cloud.githubusercontent.com/assets/16547949/23183528/6324b614-f84a-11e6-9f35-853125d9bc2c.png)